charts/kubewarden-defaults/Chart.yaml

```
@@ -22,9 +22,9 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.8.1
+version: 1.8.2-rc1
```

```suggestion
version: 1.9.0-rc1
```


I think this should be a minor bump of `version`